index.js

```diff
@@ -1,2 +1,11 @@
 'use strict';
-module.exports = process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath) || /[\\/]electron[\\/]/.test(process.execPath);
+
+const getFromEnv = () => {
+	if ('ELECTRON_IS_DEV' in process.env) {
+		return parseInt(process.env.ELECTRON_IS_DEV, 10) === 1;
```

Comment:
We could check for `!== 0` here if we wanted to accept everything but `0` as `true`.